packages/main/src/MenuItem.js

```diff
@@ -17,6 +17,28 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the <code>additionalText</code>, displayed in the end of the menu item.
+		 * @type {string}
+		 * @public
+		 * @since 1.8.0
+		 */
+		additionalText: {
+			type: String,
+		},
+
+		/**
+		 * Defines whether an additional text should be hidden.
```

Comment:
As there is not linked issue, I miss the background for adding this property - if it's requested or the team decided to add it on top.

What would be the usage - the app should detect that the app is run on phone device and set hide-additional-text. A bit more details about this property would be welcomed.

Reply:
Yes, the use case is exacly when displaying shortcuts - on mobile devices they must be hidden as irrelevant.
Some explanation added.

Reply:
If on mobile device the additional text must be hidden (I assume there is no space and it does not look nice), isn't better to just hide them always on mobile, document this behaviour in the API, and remove the property.
And if some one explicitly requests to show the additional text even on mobile (as far I can see in the issue, the mobile case is not mentioned), then we can handle it as a separate request. We can always add a new property, but removing is not that easy.

Reply:
Usage of additionalText for shortcuts is only one possible use case, so we don't want to generalize the behavior that way. That's why we decided to have second property that can hide the text.

Reply:
Ok, I see.
One more aspect that I wanted to say is that with `additionalText=""` you can have the same result  as with `hideAdditionalText=true` (unless I miss something). Is that dicsussed?

Reply:
Yes, it is possible, so we will remove this property.